libs/pika/concurrency/include/pika/concurrency/detail/contiguous_index_queue.hpp

```diff
@@ -120,6 +120,7 @@ namespace pika::concurrency::detail {
             initial_range = other.initial_range;
             current_range =
                 other.current_range.data_.load(std::memory_order_relaxed);
+            return *this;
```

Comment:
Accidentally included this change here. Opened a separate PR for it: https://github.com/pika-org/pika/pull/502.